src/styles/utilits/_variables.scss

```
@@ -0,0 +1,18 @@
+$size: 100px;
+
+$month-length: 31;
+
+$gap: 1px;
+
+$padding: 10px;
+
+$width: $size * 7 + $padding * 2 + $gap * 7;
```

1. Dont use the magic nums like 7 and 2, explain what is it with vars.
```suggestion
$width: $size * $week-length + $padding * $??? + $gap * $gaps-count;
```
2. Should we multiply gaps 7 times?))